patches/crypt_blowfish-makefile-cflags.patch

```diff
@@ -0,0 +1,11 @@
+--- Makefile	2017-08-10 14:39:36.000000000 -0700
++++ Makefile	2020-06-26 11:32:53.000000000 -0700
+@@ -18,7 +18,7 @@
+ AS = $(CC)
+ LD = $(CC)
+ RM = rm -f
+-CFLAGS = -fPIC -W -Wall -Wbad-function-cast -Wcast-align -Wcast-qual -Wmissing-prototypes -Wstrict-prototypes -Wshadow -Wundef -Wpointer-arith -O2 -fomit-frame-pointer -funroll-loops
++CFLAGS = $(CFLAG_EXTRA) -fPIC -W -Wall -Wbad-function-cast -Wcast-align -Wcast-qual -Wmissing-prototypes -Wstrict-prototypes -Wshadow -Wundef -Wpointer-arith -O2 -fomit-frame-pointer -funroll-loops
```

Comment:
I think EXTRA_CFLAGS is a more accepted name for this kind of an environment variable.